packages/components/src/FilterBar/_docs/FilterBar.mdx

```diff
@@ -105,6 +106,16 @@ Available Filter components:
   - `onSelectionChange` remains available as optional
   - **Note:** To use the subcomponents, import `FilterSelect` from `@kaizen/components` (not `@kaizen/select`)
 
+##### `isRemovable`
+
+When this attribute is applied, the Filter will become removable (a remove button will be attached to the trigger button),
+and their value will be cleared upon remove.
+
+Removable Filters will be added to the `Add Filter` menu options.
```

Comment:
Do you mean that a Filter - once removed - will be available within in the "Add Filter" menu again? If so, this could be clearer ;)

Reply:
Yes, but also by default 😬 I'll see what I can do (otherwise a suggestion would be great!)